test/system/070-build.bats

```diff
@@ -29,6 +29,15 @@ EOF
     is "$output"   "$rand_content"   "reading generated file in image"
 
     run_podman rmi build_test
-}
 
+    echo "podman build - test RUN with priv'd command"
```

Comment:
I feel awful because I didn't explain myself properly. I'm sorry, and sorry to be whining about this late on a Friday.

What I had meant was to add the 'RUN apk' to the existing dockerfile: piggyback two tests into one.

With this form, unfortunately, I think a (unlikely!) failure will cause too much confusion: that 'echo' is unlikely ever to be seen, and a user seeing the responsible command (`podman build -t ...`) may get confused as to which one it was, because there's an identical command just above. This form also doesn't give us any time savings, because it's still two podman-builds and two podman-rmis.

If you really want to do this as two separate podman-build commands, I think I'd have to recommend doing it as two separate BATS tests, the way you did it initially. With all the duplicate skip-checking and all.

I'm really sorry for not having communicated better.